targets/csharp/source/PlayFabSDK/source/Qos/QosRegionResult.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace PlayFab.QoS
+{
+    using MultiplayerModels;
+
+    public class QosRegionResult
+    {
+        public AzureRegion Region { get; set; }
+
+        public int LatencyMs { get; set; }
+
+        public int ErrorCode { get; set; }
```

Comment:
This ErrorCode is always cast from an enum QosErrorCode value, cast to an integer. Why isn't this just a QosErrorCode?

Reply:
Pf-main only expects errorcode and not string message for size concern. I am creating a Facade object here because JsonIgnore is not available here. 

---
In reply to: [288679856](https://github.com/PlayFab/SDKGenerator/pull/535#discussion_r288679856) [](ancestors = 288679856)